problems/2017-04-11-side-effects-may-include/lib/figureEmployeeIDs.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Return a list of employees with their employee ids.
+ *
+ * Hey, is this pure?
+ */
+exports.default = function(employees) {
```

Comment:
I like this especially since I can see a few ways to resolve this one which could be interesting.